### ISSUE TITLE
some-> & some->> not fontlocked

### ISSUE THREAD

**Issue by boxxxie** [clojure-emacs/clojure-mode#622]
### clojure-mode version
```
clojure-mode (version 5.14.0)
```

my work around

```lisp
(defvar clojure-core-macros
  `((,(concat
       "(\\(?:clojure.core/\\)?"
       (regexp-opt
        '(
          "some->"
          "some->>"
          ) t)
       "\\>")
     1 font-lock-keyword-face)))
```

**Comment by vemv** [comment]
It sounds worthwhile and cheap to support clojure.core macros though. Would be friendly to clojure-lsp users, for instance, I reckon.

wdyt?

**Comment by bbatsov** [comment]
@vemv Yeah, probably those we can add back to the main package. I was wondering what to do with the extra package as I guess most people don't realize that it exists.

**Comment by bbatsov** [comment]
Looking at the code we've definitely messed up something as some macros are in the main package and some in the secondary.

**Comment by bbatsov** [comment]
I guess we'll need to check how many of the things that are listed as `clojure.core` built-ins in the extra package are actually macros and move them to `clojure-mode`. I'll reopen this ticket.

**Comment by boxxxie** [comment]
point me to where i should look in the .el file, and i can compile a list of the missing macros from clojure.core

**Comment by bbatsov** [comment]
See https://github.com/clojure-emacs/clojure-mode/blob/master/clojure-mode.el#L843

**Comment by boxxxie** [comment]
macros i left out are from pprint_base.clj cl_format.clj template.clj core_proxy.clj tap.clj test.clj shell.clj junit.clj repl.clj genclass.clj main.clj

**Comment by vemv** [comment]
Released as clojure-mode 5.15.0, which will be available within a couple hours
